index.bs

```diff
@@ -322,6 +318,19 @@ it should change the DNS-SD service name to a new value, indicating a new
 mechanism for metadata discovery.
 
 
+Computing the Agent Fingerprint {#computing-agent-fingerprint}
+-------------------------------
+
+The <dfn>agent fingerprint</dfn> of an agent is computed by following these
+steps:
+
+1. Compute the [[RFC7469#section-2.4|SKPI Fingerprint]] of the [=agent certificate=]
+    according to [[!RFC7469]] using [[RFC6234|SHA-256]] as the hash algorithm.
```

Comment:
Is this "The Pin Directive" described in that spec?

https://datatracker.ietf.org/doc/html/rfc7469#section-2.1.1

Reply:
Not exactly.  This is just referring to the SPKI which the pin directive consumes.  The pin directives are for use in HTTP server headers to pin certificates to hosts.